Downstream-only patch to refer to /run instead of the legacy /var/run
directory in the downstream nscd systemd socket file.

diff --git a/nscd/nscd.socket b/nscd/nscd.socket
index 7e512d5339fa1136..52a67608c7c55475 100644
--- a/nscd/nscd.socket
+++ b/nscd/nscd.socket
@@ -2,7 +2,7 @@
 Description=Name Service Cache Daemon Socket
 
 [Socket]
-ListenDatagram=/var/run/nscd/socket
+ListenDatagram=/run/nscd/socket
 
 [Install]
 WantedBy=sockets.target
